avro-converter/src/main/java/io/confluent/connect/avro/AvroData.java

```diff
@@ -500,6 +503,27 @@ private static Object fromConnectData(Schema schema, org.apache.avro.Schema avro
     }
   }
 
+  private static org.apache.avro.Schema avroSchemaForUnderlyingMapEntryType(
```

Comment:
I know this is a private method, but how about JavaDoc on this method that explains it does and how it differs from `avroSchemaForUnderlyingTypeIfOptional `?